### PR TITLE
Navigation transition added

### PR DESCRIPTION
**Pull request by giteshk11**
# Issue Being Addressed
#173
Put the issue(s) # that your PR is addressing here. e.g. #15, #29

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description
There's a slight delay when we click on clients or back button not sure why it's happening.
**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?

**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?

**For New Features:** What feature are we adding? Explain your technical approach to doing this.

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

**For Other:** State what 'type' of PR yours is and elaborate on the PR's technical content.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Log in as '...'
2. Go to '...'
3. Click on '...'
4. See the updated element x

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

**Comment by paulchrisluke** [comment]
@mort3za "There's a slight delay when we click on clients or back button not sure why it's happening."

**Comment by paulchrisluke** [comment]
every page is swiping, this should only occur on the back button presses

**Comment by paulchrisluke** [comment]
I'm expecting this animation when I hit back when viewing an individual profile as well here: https://deploy-preview-205--browtricks.netlify.app/tenant/salon-2-95/clients/48/edit

please do not trigger this animation on app load, like it is currently. (initial page load swipes in)

File: src/App.vue [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
-        v-if="overlayModel.title || overlayModel.message"
-        class="w-full h-full fixed block top-0
-        left-0 z-110"
+  <div id="app" class="min-h-vh100 h-full text-center font-sans">
+    <transition :name="transitionName">
+      <component
+        :is="this.$route.meta.layout || 'div'"
+        class="h-full min-h-vh100"
       >
-        <BaseOverlaySuccess
```
Comment (mort3za) [inline src/App.vue:18]: The success overlay is removed, restore it please. We need to show an overlay screen on certain actions like save client in client edit page.

Comment (giteshk11) [reply src/App.vue:18]: is it available in staging?

Comment (mort3za) [reply src/App.vue:18]: Yes

**Review by paulchrisluke — APPROVED** [review]
ux looks good

**Pull request merged**